src/test/java/RecipeTest.java

```diff
@@ -35,18 +37,20 @@
 
   @Before
   public void setup() {
-    recipe = new Recipe(NAME, DESCRIPTION, new LinkedList(STEPS));
+    recipe = new Recipe(NAME, DESCRIPTION, TAGS, INGREDIENTS, new LinkedList(STEPS));
   }
 
   @Test
   public void appendStep() {
-    List<Step> expectedSteps = new LinkedList<>(STEPS);
-    expectedSteps.add(new Step("butter the bread"));
-
-    Recipe expected = new Recipe(NAME, DESCRIPTION, expectedSteps);
-    recipe.appendStep(new Step("butter the bread"));
+    List<Step> expectedSteps = Arrays.asList(
```

Comment:
Optional: you could copy STEPS into expected steps and then add the new step, just like you were doing before.

Reply:
I thought about doing that, but I like how creating expectedSteps in this way shows the exact order of the steps, which is useful in the later methods that test inserting a step in the middle of the list.